### PR TITLE
from_json support non-existence fields in json str

### PR DESCRIPTION
**Pull request by lovio**
1.  from_json support non-existence fields in json str.
2. Suppress UB errors caused by system-installed rapidjson by using rapidjson imported via git submodule
FIX below errors.

/usr/include/rapidjson/internal/stack.h:117:13: runtime error: applying non-zero offset 16 to null pointer SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /usr/include/rapidjson/internal/stack.h:117:13 in /usr/include/rapidjson/internal/stack.h:117:13: runtime error: applying non-zero offset 1 to null pointer SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /usr/include/rapidjson/internal/stack.h:117:13 in /usr/include/rapidjson/internal/stack.h:117:13: runtime error: applying non-zero offset 16 to null pointer SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /usr/include/rapidjson/internal/stack.h:117:13 in

### CODE REVIEW

File: include/generator_cpp_fixture.h [hunk 1]
```diff
@@ -6562,7 +6562,8 @@ struct NodeReader
         // Try to find a member with the given key
         rapidjson::Value::ConstMemberIterator member = json.FindMember(key);
         if (member == json.MemberEnd())
-            return false;
+            // omitted field in json str is allowed.
```
Comment (hurricane1026) [inline include/generator_cpp_fixture.h:6565]: 这样，是不是并不限制在不在末尾？

**Comment by lovio** [comment]
> 那么 stdb 也要引入 rapidjson 么？

stdb已经引入了

**Comment by hurricane1026** [comment]
> 那么 stdb 也要引入 rapidjson 么？

好吧，已经引入了，要避免 stdb 编译的时候同时用两份 rapidjson 了

**Comment by deafwolf** [comment]
这么直接 add_subdirectory，能避免重复引入么？

**Comment by lovio** [comment]
> 这么直接 add_subdirectory，能避免重复引入么？

项目里没有引入 fbe

**Pull request merged**